packages/frontend/src/system-integration/notifications.ts

```diff
@@ -81,11 +115,50 @@ async function showNotification(
     try {
       const notificationInfo =
         await BackendRemote.rpc.getMessageNotificationInfo(accountId, messageId)
-      const { chatName, summaryPrefix, summaryText } = notificationInfo
+      let summaryPrefix = notificationInfo.summaryPrefix
+      const summaryText = eventText ?? notificationInfo.summaryText
+      const chatName = notificationInfo.chatName
+      let icon = getNotificationIcon(notificationInfo)
+      if (isWebxdcInfo && chatId === -1) {
```

Comment:
why is chatID `-1` here?

Reply:
ah because it isn't there yet.... I'd rather use `null` or `undefined` to represent that state.
Or add the chat id to the `IncomingWebxdcNotify` event in core.

Reply:
thinking about it more it makes sense to add it to the event. It already has contact id.

https://github.com/deltachat/deltachat-core-rust/blob/191eb7efdd2765af0b9440a1316e850ed321b2b2/src/events/payload.rs#L111-L123